src/form.js

```diff
@@ -33,11 +34,22 @@ export default class SimpleteForm extends HTMLElement {
 		let field = this.searchField;
 		field.setAttribute("autocomplete", "off");
 
+		this.addAriaRoles();
+
 		let onQuery = debounce(this.queryDelay, this, this.onQuery);
 		this.addEventListener("input", onQuery);
 		this.addEventListener("change", onQuery);
 		this.addEventListener("simplete-suggestion-selection", this.onSelect);
 		field.addEventListener("keydown", this.onInput);
+		this.addEventListener("simplete-suggestion-toggle", this.onToggle);
+	}
+
+	addAriaRoles() {
+		this.searchField.setAttribute("role", "combobox");
+		this.searchField.setAttribute("aria-autocomplete", "list");
+		this.searchField.setAttribute("aria-expanded", "false");
+		this.suggestions.id = this.suggestions.id || "simplete-suggestions-" + nid();
```

Comment:
Why not move this into `suggestions.js`? Seems weird to control another element's ID.